### ISSUE TITLE
Console execution and queuing for large reports

### ISSUE THREAD

**Issue by iamkeir** [barrelstrength/sprout#21]
Large reports, even when as optimal as possible, can be a pain due to:

1. Craft CP being "locked up" whilst running report
2. HTTP request limits/timeouts

Could this potentially be improved if reports were "requested", but then queued and executed via the console - perhaps with some form of "report completed" notification? Queued console commands are not subject to the same timeout limitations.

**Comment by BenParizek** [comment]
Good idea. I'll add this as an FR to explore once we get upgraded to Craft 4.

**Comment by hassanmah** [comment]
@BenParizek was this ever implemented? I have large datasets, that struggle/timeout using the cp
